lib/elixir/lib/supervisor/spec.ex

```diff
@@ -75,6 +75,10 @@ defmodule Supervisor.Spec do
     * `:transient` - the child process is restarted only if it
       terminates abnormally, i.e., with an exit reason other than
       `:normal`, `:shutdown` or `{:shutdown, term}`
+  
+  Notice that whenever supervisor with `:transient` restart value reaches `max_restart_intensity`,
```

Comment:
It is great point to add but I think we need to be extra clear that is the child specification of the supervisor that lists `:transient` and not a `transient` restart value in the child specifications of the supervisor.